src/clients/dotnet/README.md

```diff
@@ -270,8 +270,7 @@ for (int i = 0; i < transfers.Length; i += BATCH_SIZE)
     {
         batchSize = transfers.Length - i;
     }
-    var segment = new ArraySegment<Transfer>(transfers, i, batchSize);
-    createTransfersError = client.CreateTransfers(segment.Array);
+    createTransfersError = client.CreateTransfers(transfers[i..batchSize]);
```

Comment:
This isn't correct--the range syntax in C# is of the form [inclusive_start..exclusive_end] (while in the ArraySegment ctor of the previous version the second int is a count).

The line should be written: `createTransfersError = client.CreateTransfers(transfers[i..(i + batchSize)]);`

Reply:
Thanks @tenatus!
Ref https://github.com/tigerbeetle/tigerbeetle/pull/2458